udata/core/dataset/models.py

```diff
@@ -324,7 +322,11 @@ def quality(self):
             * description length
             * and so on
         """
+        from udata.models import Discussion
```

Comment:
The computation has been moved to `compute_quality_score` and `check_availability`, we are just assembling the dict here. But I'm open to better suggestions, maybe it should be done completely asynchronously, or even externalized…